region/crateria/central.json

```diff
@@ -5112,8 +5151,13 @@
                   "name": "Crateria Supers G-Mode Up with PBs",
                   "notable": true,
                   "requires": [
+		    {"itemNotCollectedAtNode": 3},
                     "canConsecutiveWalljump",
                     "Morph",
+		    {"or": [
+		      "canRiskPermanentLossOfAccess",
+		      "canRightFacingDoorXRayClimb"
```

Comment:
Adding this option is pretty sketchy.
Keep in mind direct GMode puts you at 4 health, so they need grapple or spacejump to cross the spike pit already so its better to rely on the xray climb strat as the way up here.

Reply:
Adding this is a way to avoid `canRiskPermanentLossOfAccess`. This could put it in logic if the item in that location is grapple, space jump, e-tank, or ammo to CF.

Reply:
I think this is negative E.V.
With the extra xray climb requirement, its not likely to break anything.  Its just a ton of requirements to get around a spike hit.  
It also doesn't follow the get item return to start rule 😢 /s

The model would like to have this added, however. So we'd want a schema object requiring the 2->1xray climb strat be possible as a requirement of this strat, I think.  It wouldn't be wrong to more broadly require a path from 2->1 be available even if you must leave the room, since it is 1->4 that this strat is avoiding.